framework/src/play/test/FunctionalTest.java

```diff
@@ -187,27 +187,38 @@ public static Response POST(Request request, Object url, Map<String, String> par
 
         for (String key : files.keySet()) {
             Part filePart;
-            try {
-                filePart = new FilePart(key, files.get(key));
-            } catch (FileNotFoundException e) {
-                throw new RuntimeException(e);
-            }
+            filePart = new FilePart(key, files.get(key));
             parts.add(filePart);
         }
 
-        MultipartRequestEntity requestEntity = new MultipartRequestEntity(parts.toArray(new Part[parts.size()]), new FluentCaseInsensitiveStringsMap()); 
-        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+        MultipartBody requestEntity = null;
```

Comment:
I am curious to know why you use **MultipartBody** and remove **MultipartRequestEntity**?